client/Makefile

```diff
@@ -34,17 +34,15 @@ start-frontend:
 
 .PHONY: e2e
 e2e:
-	node node_modules/jest/bin/jest.js \
+	APP_PORT=${CXG_SERVER_PORT} \
```

Comment:
@mweiden Ahh I see what you're saying now! Before I had:

```
CXG_SERVER_PORT=${CXG_SERVER_PORT} \
```

Which is confusing, because I really wanted to set which port Puppeteer should hit, instead of overwriting the default `CXG_SERVER_PORT` value

So I changed the name to `APP_PORT` now, and using it in `client/__tests__/e2e/config.js` like so:

```ts
export const appPort = process.env.APP_PORT || ENV_DEFAULT.CXG_CLIENT_PORT;
```

Which means we can change the makefile script here to be:

```
APP_PORT=${CXG_SERVER_PORT} \
```

Does it look better now? Thank you!

Reply:
nit: Given that you are now bringing the environment into `config.js` via `environment.default.json`, isn't it possible to remove the `APP_PORT` variable entirely and have two variables/concepts (`CXG_SERVER_PORT` and `CXG_CLIENT_PORT`) instead of three?

in `config.js`, this could be:
```
export const clientPort = process.env.CXG_CLIENT_PORT || ENV_DEFAULT.CXG_CLIENT_PORT;
export const appUrlBase = process.env.CXG_URL_BASE || `http://localhost:${clientPort}`;
```

This change would also make your make target simpler.